### DISCUSSION TITLE
Let's Encrypt Support

### DISCUSSION THREAD

**Discussion by nealhead** [ZoeyVid/NPMplus#1134] [Q&A] [answered]
I just wanted to see if I could get some clarification on the most recent update:

> NOTE: lets encrypt wants to remove ocsp support, because of this NPMplus may change its default CA to zerossl in the future

It's my understanding that they are removing OCSP support for security reasons and instead recommend CRLs. Is this not possible with your software?

I am worried about the switch to ZeroSSL, don't they charge for wildcard SSL?

**Comment by Zoey2936** [comment, marked as answer]
No they are removing ocsp because it is very expensive for them to keep ocsp running. CRL is useless they will never contain all certs/be updated in real time.

Zersoll supports free wildcard certs via acme.
